packages/amplication-client/src/VersionControl/BuildHeader.tsx

```diff
@@ -1,39 +0,0 @@
-import React from "react";
-import { Link } from "react-router-dom";
-
-import * as models from "../models";
-import { ClickableId } from "../Components/ClickableId";
-
-import "./BuildHeader.scss";
-
-const CLASS_NAME = "build-header";
-
-type Props = {
-  build: models.Build;
-  isError: boolean;
-};
-
-const BuildHeader = ({ build, isError }: Props) => {
-  const deployedClassName = `${CLASS_NAME}--deployed`;
-
-  return (
-    <div className={`${CLASS_NAME} ${deployedClassName} `}>
-      <ClickableId
-        to={`/${build.resourceId}/builds/${build.id}`}
-        id={build.id}
-        label="Build ID"
-        eventData={{
-          eventName: "buildHeaderIdClick",
-        }}
-      />
-      {isError ? (
-        <Link to={`/${build.resourceId}/builds/${build.id}`}>
-          <h3 className="error-message">Build Failed Check Logs</h3>
-        </Link>
-      ) : null}
-      <span className="spacer" />
-    </div>
-  );
-};
-
-export default BuildHeader;
```

Comment:
Was this change made on purpose? I cannot see any relationship with the issue

Reply:
right, but this is an unused component so I deleted it (same for the others...)

Reply:
I agree with @overbit question - this is the type of changes we should do on a separate PR to have better visibility in the future on the reasons that trigger each change. (I would also separate it to different PRs one for refactoring of the enum and another one for adding the new event)

Reply:
but it doesn't make sense to create an event key in the new `enum` for a component that is not in use anymore
(line 25)
```
eventData={{
   eventName: "buildHeaderIdClick",
}}
```
regarding the separation into 2 PRs, I understand...next time 😄

Reply:
Well, it is in is being used if the file is still there :-) anyway that was just a comment for next time

Reply:
I'm happy with just raising the awareness for this PR. We will try to make PR more issue specific as we go.